docs/gitlab-pipelines.md

```diff
@@ -0,0 +1,249 @@
+
+   ![GitLab Settings](assets/gitlab-settings.png)
+
+Then select __Variables__, followed by the __Add Variable__ button:
+
+   ![GitLab Variables](assets/gitlab-variables.png)
+   
+When you create the [CI/CD configuration file](#create-the-gitlab-configuration), your Sauce Labs authentication variables appear like so:
+
+```yaml
+variables:
+  SAUCE_USERNAME: ${SAUCE_USERNAME}
+  SAUCE_ACCESS_KEY: ${SAUCE_ACCESS_KEY}
+```
+
+## Create the Saucectl Configuration
+
+Create the `.sauce` directory at the root of your project and add a `config.yaml` file that points [`saucectl`](cli-reference.md) to your existing `tests` directory. 
+
+With the `suites` field you can specify a group of tests as well as the browser `settings` you wish to use.
+
+Below are some examples:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--puppeteer-->
+
+```yaml
+# ./.sauce/puppeteer.yml
+apiVersion: v1alpha
+metadata:
+  name: Testing Puppeteer Support
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests
+suites:
+  - name: "chrome"
+    match: ".*.(spec|test).js$"
+    settings:
+      browserName: "chrome"
+image:
+  base: saucelabs/stt-puppeteer-jest-node
+  version: v0.1.8
+sauce:
+  region: us-west-1
+```
+
+<!--playwright-->
+
+```yaml
+# ./.sauce/playwright.yml
+apiVersion: v1alpha
+metadata:
+  name: Testing Playwright Support
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests
+suites:
+  - name: "chrome"
+    match: ".*.(spec|test).js$"
+    settings:
+      browserName: "chrome"
+image:
+  base: saucelabs/stt-playwright-jest-node
+  version: v0.1.9
+sauce:
+  region: us-west-1
+```
+
+<!--testcafe-->
+
+```yaml
+# ./.sauce/testcafe.yml
+apiVersion: v1alpha
+metadata:
+  name: Testing TestCafe Support
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests
+suites:
+  - name: "chrome"
+    match: ".*.(spec|test).js$"
+    settings:
+      browserName: "chrome"
+image:
+  base: saucelabs/stt-testcafe-node
+  version: v0.1.7
+sauce:
+  region: us-west-1
+```
+
+<!--cypress-->
+
+```yaml
+# ./.sauce/cypress.yml
+apiVersion: v1alpha
+metadata:
+  name: Testing Cypress Support
+  tags:
+    - e2e
+    - release team
+    - other tag
+  build: Release $CI_COMMIT_SHORT_SHA
+files:
+  - ./tests
+suites:
+  - name: "chrome"
+    match: ".*.(spec|test).js$"
+    settings:
+      browserName: "chrome"
+image:
+  base: saucelabs/stt-cypress-mocha-node
+  version: v0.1.11
+sauce:
+  region: us-west-1
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Create the GitLab Configuration
+
+In the root of your project, create a `gitlab-ci.yaml` file. Below are examples in the supported Testrunner Toolkit test frameworks:
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--puppeteer-->
+
+```yaml
+image: saucelabs/stt-puppeteer-jest-node:latest
```

Comment:
We should mention somewhere that we discourage the use of `latest` as a tag. It sure is convenient, but many users just leave it like this in their config and due to docker caching the tag, they get mislead by the image not actually representing a latest release later on (unless the image is re-pulled).

Reply:
I just made sure the version tags matched the `config.yml` numbers for now, not problematic until those numbers drift considerably.